src/pipx/venv_inspect.py

```diff
@@ -99,19 +107,31 @@ def get_apps(dist: metadata.Distribution, bin_path: Path) -> List[str]:
         entry = line.split(",")[0]  # noqa: T484
         inst_file_path = Path(dist.locate_file(entry)).resolve()
         try:
-            if inst_file_path.parent.samefile(bin_path):
-                apps.add(inst_file_path.name)
+            if inst_file_path.parent.samefile(resource_path):
+                names.add(inst_file_path.name)
         except FileNotFoundError:
             pass
 
-    return sorted(apps)
+    return sorted(names)
+
+
+def get_man_pages(dist: metadata.Distribution, man_path: Path) -> List[str]:
+    man_pages = []
+    for i in range(1, 10):
+        man_section = "man%d" % i
+        names = get_resources("man", dist, man_path / man_section)
+        for name in names:
+            man_pages += [str(Path(man_section) / name)]
+    return man_pages
```

Comment:
OK, I have changed that in the latest commit (49b13fb40c9057b5f67aa15d6901d8a7b22b0399). Also I have created a new constant `MAN_SECTIONS` and changed the loops to iterate over `MAN_SECTIONS` instead of the more complicated way with `range(1, 10)`.